plugins/inventory/now.py

```diff
@@ -722,10 +414,9 @@ def parse(self, inventory, loader, path, cache=True):
             )
 
         # TODO: Insert caching here once we remove deprecated functionality
```

Comment:
I think we need to address this `TODO`.

Reply:
@Akasurde I would make it a separate pull request.